docs/ServoParameters.md

```diff
@@ -0,0 +1,18 @@
+#ServoParameters
+
+Each PhidgetServo will have its own set of parameters. Documentation for [PhidgetServos can be seen here](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Servo.md),a full list of PhidgetServo products [here](http://www.phidgets.com/products.php?category=10), and a full servo primer guide [here](http://www.phidgets.com/docs/Servo_Motor_and_Controller_Primer). 
+
+##Methods
+
+###Series 3000-3499 Servos
```

Comment:
Should not specify the servo numbers here as the parameters class should automatically handle this. The user shouldn't have to look up their model to figure out how to use our class. It will be a uniform implementation across the board with any servo making it clear what it supports.